function-support.html

```diff
@@ -813,105 +832,70 @@ <h3 id="other-letters">Other Letters</h3>
+				<td><span class="katex"><span class="mathbf">A-Z a-z 0-9</span></span></td>
+				<td rowspan="5">&nbsp;&nbsp;&nbsp;</td>
+				<td>Double-struck</td>
+				<td><span class="katex"><span class="mathbb">A-Z k</span></span></td>
 			</tr>
 			<tr>
-				<td>Devanagari</td>
-				<td>0900 – 097F</td>
-				<td>Tibetan</td>
-				<td>0F00 – 0FFF</td>
+				<td>Italic</td>
+				<td><span class="katex"><span class="mathit">A-Z a-z</span></span></td>
+				<td>Sans serif</td>
+				<td><span class="katex"><span class="textsf">A-Z a-z 0-9</span></span></td>
 			</tr>
 			<tr>
-				<td>Bengali</td>
-				<td>0980 – 09FF</td>
-				<td>Georgian</td>
-				<td>10A0 – 10FF</td>
+				<td>Bold Italic</td>
+				<td><span class="katex"><span class="boldsymbol">A-Z a-z</span></span></td>
+				<td>Sans serif bold</td>
+				<td><span class="katex"><span class="textbf textsf">A-Z a-z 0-9</span></span></td>
 			</tr>
 			<tr>
-				<td>Gurmukhi</td>
-				<td>0A00 – 0A7F</td>
-				<td>CJK symbols and punctuation</td>
-				<td>3000 – 303F</td>
+				<td>Script</td>
+				<td><span class="katex"><span class="mathscr">A-Z</span></span></td>
+				<td>Sans serif italic</td>
+				<td><span class="katex"><span class="textit textsf">A-Z a-z</span></span></td>
 			</tr>
 			<tr>
-				<td>Gujarati</td>
-				<td>0A80 – 0AFF</td>
-				<td>Hiragana</td>
-				<td>3040 – 309F</td>
-			</tr>
-			<tr>
-				<td>Oriya</td>
-				<td>0B00 – 0B7F</td>
-				<td>Katakana</td>
-				<td>30A0 – 30FF</td>
-			</tr>
-			<tr>
-				<td>Tamil</td>
-				<td>0B80 – 0BFF</td>
-				<td>CJK ideograms</td>
-				<td>4E00 – 9FAF</td>
-			</tr>
-			<tr>
-				<td>Telugu</td>
-				<td>0C00 – 0C7F</td>
-				<td>Hangul</td>
-				<td>AC00 – D7AF</td>
-			</tr>
-			<tr>
-				<td>Kannada</td>
-				<td>0C80 – 0CFF</td>
-				<td>Full width punctuation</td>
-				<td>FF00 – FF60</td>
+				<td>Fractur</td>
+				<td><span class="katex"><span class="mathfrak">A-Z a-z</span></span></td>
+				<td>Monospace</td>
+				<td><span class="katex"><span class="mathtt">A-Z a-z 0-9</span></span></td>
 			</tr>
 		</tbody>
 	</table>
-	
+
+	<h4 id="letters-in-text">Letters inside \text{}</h4>
+		<p><code>\text{…}</code> will accept functions: <code>\aa</code> <code>\AA</code> <code>\ae</code> <code>\AE</code> <code>\oe</code> <code>\OE</code> <code>\o</code> <code>\O</code> <code>\ss</code> <code>\i</code> <code>\j</code><br>
+		… and return them as: \(\text{\aa\: \AA\: \ae\: \AE\: \oe\: \OE\: \o\: \O\: \ss\: \i\: \j}\)</p>
+
+	<h3 id="unicode">Unicode</h3>
+		<p>The letters listed above will render in any KaTeX rendering mode.</p>
+		
+		<p>If the KaTeX rendering mode is set to <code>strict: false</code> or <code>strict:"warn"</code> (default), then KaTeX will accept all Unicode letters.</p>
+		
+		<p>If a letter is not listed above, it may cause small vertical alignment issues. KaTeX has detailed metrics for glyphs in Latin, Greek, and Cyrillic, but other glyphs are treated as if they are each as tall as the letter <q>M</q>.</p>

```

Comment:
I would merge the above two paragraphs.  And maybe also stress that unlisted symbols won't be in the fonts, so they fall back to an unknown system font, and won't "look right", in addition to the metrics issue.